### ISSUE TITLE
No explicit button for the 'home page'

### ISSUE THREAD

**Issue by ixxie** [vector-im/element-web#15246]
### Description

Element has what I would call a 'home page' where I can see people and rooms I subscribe to which are not part of a community. The navigation, however, offers no explicit button to reach this central page.

Here is how the navigation looks like when a community is selected:

![Screenshot from 2020-09-20 22-46-20](https://user-images.githubusercontent.com/20320695/93720785-ec36cc80-fb93-11ea-8f43-7ea29051216f.png)

In order to navigate back to the home page, one must deselect the community by pressing it again:

![Screenshot from 2020-09-20 22-46-25](https://user-images.githubusercontent.com/20320695/93720791-ee992680-fb93-11ea-9453-70fe2fd52f89.png)

This is rather intuitive; before figuring this out, I tried to press my own username in the top left; this turns out to do exactly the same thing as the overflow menu to the right of my username (seems strange, but not completely implausible). I spend some minutes looking around for a home page button until I figured out it didn't exist.

I would love an explicit 'home' button on the top of the community bar. Alternatively, clicking my user name navigate to the home page would also be more intuitive.

_Sidenote:_ its also a little strange that the main chat doesn't change when navigating between communities; I think it makes it hard to localize where you are at in navigation. The community in this example has no rooms, but even having a placeholder landing page with some basic information about the community would be more intuitive.

### Version information

- **Platform**: Desktop
- **OS**: NixOS (unstable channel)
- **Version**:
  * Element version: 1.7.5
  * olm version: 3.1.3

**Comment by jryans** [comment]
Thanks for the feedback, you're not alone in finding the communities navigation very strange. It's definitely on the list to be reworked.

**Comment by stephanerosa** [comment]
I'm new to Element, coming from the world of Discord. I'm totally lost with the navigation (and many other things but ok). To leverage on the great suggestions from @ixxie above, I add some more ideas.

Really the concept of communities is totally useless at the moment, at least until the following is fixed:

- There is no way to auto_join_communities in settings. If there was, we could at least auto invite newcomers to a Welcome community with a nice home page and give them the first instructions to survive the quite poor UX
- There is no way to /join a community, nor to actually find a community at all even if public.
- The community homepage is great except nobody will ever find it, its link being buried in a tiny three dots link
- Inviting people to a community seems not to work. Only after they are ALSO invited to one of the community rooms, then they get both notifications (for community and room). But inviting only to the community does not do anything. Doing it the other way around (first room, then community) does not work either... until you disinvite for the room and suddenly the community invite appears to the user.

I suggest some quick wins:

- clicking on a community button ALSO brings up the Community page, so you know where you are. At least, do it every first time in a session, then remember the last room the user was in.
- remove this on/off "feature" when clicking on communities (once you get the community, the other you get the "home") as it's horribly confusing. The Home should be when clicking on the user icon, or better a dedicated top home button.

/me thinks this utterly urgent. Actually the whole UX is utterly urgent as even with great features if you drive users nuts they won't stay :-(

**Comment by Casuallynoted** [comment]
I came here specifically to agree with this- I’ve been hosting a Synapse server for some time now, and every time I have a friend try to join it using Element the UX/UI is what makes them immediately stop. Coming from a service like Discord or Slack where things are segmented into servers with their own unique channels, the layout of Element feels archaic. DMs are treated exactly the same as rooms and communities both in the UI and seemingly by the app itself, and it’s super confusing to the user. It feels like you’re doing math problems trying to figure out where stuff is. I don’t mean to sound overly critical because I actually *really* love the work that you guys are doing with Element/Riot, but especially on mobile it’s too much work on the user’s part to enjoy using.

**Comment by t3chguy** [comment]
> Element has what I would call a 'home page' where I can see people and rooms I subscribe to which are not part of a community. The navigation, however, offers no explicit button to reach this central page.

This is coming as part of spaces

**Comment by t3chguy** [comment]
Closing in favour of https://github.com/vector-im/element-web/issues/17203
